pyfar/dsp/dsp.py

```diff
@@ -1498,3 +1498,144 @@ def decibel(signal, domain='freq', log_prefix=None, log_reference=1,
+    """
+    # check input
+    if not isinstance(signal, pyfar.Signal):
+        raise ValueError(f"signal is type '{signal.__class__}'"
+                         " but must be of type 'Signal'.")
+    # cumpute energy data
+    data = np.sum(signal.time**2, axis=-1)
+    if keepdims:
+        data = data[..., None]
+    return data
+
+
+def power(signal, keepdims=False):
+    r"""
+    Compute the power of a signal.
+
+    The power is calculated as
+
+    .. math::
+
+        \frac{1}{N}\sum_{n=0}^{N-1}|x[n]|^2
+
+    based on the time data for each channel separately.
+
+    Parameters
+    ----------
+    signal : Signal
+        The signal to compute the power from.
+    keepdims : bool, optional
+        If ``False``, the shape of the returned array equals ``signal.cshape``.
+        If ``True``, an axis will be appended, so it's broadcastable to the
+        shape of ``signal.time`` and ``signal.freq``.
+        The default is ``False``.
+
+    Returns
+    -------
+    data : numpy.ndarray
+        The channel-wise power of the input signal.
+
+    Notes
+    -----
+    Due to the calculation based on the time data, the returned power is
+    independent of the signal's ``fft_norm``.
+    The power equals the squared RMS of a signal. :py:func:`~pyfar.dsp.energy`
+    and :py:func:`~pyfar.dsp.rms` can be used to compute the energy and the
+    RMS.
+    """
+    # check input
+    if not isinstance(signal, pyfar.Signal):
+        raise ValueError(f"signal is type '{signal.__class__}'"
+                         " but must be of type 'Signal'.")
+    # cumpute power data
+    data = np.sum(signal.time**2, axis=-1)/signal.n_samples
+    if keepdims:
+        data = data[..., None]
+    return data
+
+
+def rms(signal, keepdims=False):
+    r"""
+    Compute the root mean square (RMS) of a signal.
+
+    The RMS is calculated as
+
+    .. math::
+
+        \sqrt{\frac{1}{N}\sum_{n=0}^{N-1}|x[n]|^2}
+
+    based on the time data for each channel separately.
+
+    Parameters
+    ----------
+    signal : Signal
+        The signal to compute the RMS from.
+    keepdims : bool, optional
+        If ``False``, the shape of the returned array equals ``signal.cshape``.
+        If ``True``, an axis will be appended, so it's broadcastable to the
+        shape of ``signal.time`` and ``signal.freq``.
+        The default is ``False``.
+
+    Returns
+    -------
+    data : numpy.ndarray
+        The channel-wise RMS of the input signal.
+
+    Notes
+    -----
+    The RMS equals the square root of the signal's power.
+    :py:func:`~pyfar.dsp.energy` and :py:func:`~pyfar.dsp.power` can be used
+    to compute the energy and the power.
+    """
+    # check input
+    if not isinstance(signal, pyfar.Signal):
+        raise ValueError(f"signal is type '{signal.__class__}'"
+                         " but must be of type 'Signal'.")
+    # cumpute rms data
+    data = np.sqrt(np.sum(signal.time**2, axis=-1)/signal.n_samples)
```

Comment:
As you say in the note: sqrt(power), so why not do something like this:

```suggestion
    data = np.sqrt(power(signal))
```